src/views/docs/Team.tsx

```diff
@@ -67,6 +67,7 @@ export class Team extends React.PureComponent<{}, any> {
         const jp = getLanguageFlag("japan", country, "jp");
         const fi = getLanguageFlag("finnish", country, "fi");
         const it = getLanguageFlag("italian", country, "it");
+        const un = getLanguageFlag("un", country, "un");
```

Comment:
This variable (`un`) isn't used since you just use a string literal (`"un"`, with quotes) below.  You can just delete this line and I think it will work as expected 😃

Reply:
Thank you, I'll try it now :)